addon/components/polaris-choice-list.js

```diff
@@ -104,6 +121,22 @@ export default Component.extend({
    */
   onChange() {},
 
+  /**
+   * @private
+   */
+  hasError: bool('error'),
```

Comment:
Not sure it's a big deal, but the React implementation has this as `error != null` so maybe we should look at mimicking that behaviour here?